Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -26,12 +26,12 @@ ly_add_target(
             ${pal_dir}
         PRIVATE
             ${common_dir}
-            AZ::AzNetworking
+            AZ::AzNetworking.API
```

Comment:
The INCLUDE_DIRECTORIES doesn't accept a CMake Target.
The AzFramework would get access to the AZ::AzNetworking.API include directories via the BUILD_DEPENDENCIES.